### ISSUE TITLE
how to mix two image about rgb and gray?

### ISSUE THREAD

**Issue by cena001plus** [argusswift/YOLOv4-pytorch#206]
Cascade a three channel RGB and a single channel grayscale image as an image input, and correspond to the same label. The scale and content of the two images are the same. How to modify the input?

**Comment by argusswift** [comment]
> Cascade a three channel RGB and a single channel grayscale image as an image input, and correspond to the same label. The scale and content of the two images are the same. How to modify the input?

emmm,that makes a challenge for me.
You need to modify the image input in the data-processing firstly(set three channels to four channels), then modify the original data input part into four channels in the dataloader, and finally set the number of input channels to 4 in the head conv of the network
